elado/malom/index.md

```diff
@@ -3,4 +3,56 @@ layout: elado
 title: Eladó malom
```

Comment:
Ja, esetleg ezt is átírhatod. Ez a `title` jelenik meg a böngészőben az oldal fülén.

Ha át akarod írni, azt előlről kell megint csinálnod, az én repositorymból kiindulva. Ez a változtatás már "merge-elve" van. Láthatod is az élő weboldalon: http://www.danieldarabos.com/megingat/elado/malom/